packages/ingest/src/granule.ts

```diff
@@ -200,13 +258,23 @@ const getNameOfFile = (file) => file.fileName || file.name;
  *    file: file object
  *  }
  */
-function generateMoveFileParams(sourceFiles, destinations) {
+export function generateMoveFileParams(
+  sourceFiles: File[],
+  destinations: {
+    bucket: string,
+    filepath?: string,
+    regex: string | RegExp
+  }[]
+): MoveFileParams[] {
   return sourceFiles.map((file) => {
     const fileName = getNameOfFile(file);
+
+    if (fileName === undefined) return { file };
+
     const destination = destinations.find((dest) => fileName.match(dest.regex));
 
     // if there's no match, we skip the file
-    if (!destination) return { source: null, target: null, file };
+    if (!destination) return { file };
```

Comment:
Is this a breaking change?

Reply:
It is, noted here: https://github.com/nasa/cumulus/pull/1798/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR16